play/heatdist.cc

```diff
@@ -1,93 +1,106 @@
-#include <mpi.h>
```

Comment:
This looks like you ran clang-format over this file (except one change to the grid size).  Can we revert this formatting change and replace the change to the grid size with a macro as described later in my review?